src/Utils.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+var Matrix = require('ml-matrix');
+
+class Utils {
+    /**
+     * Apply matrix sum between two matrices and retrieves the result over the first argument matrix.
+     * @param {Matrix} A
+     * @param {Matrix} B
+     * @return {Matrix} A
+     */
+    static matrixSum(A, B) {
```

Comment:
They are currently not documented but they exist.
See [here](https://github.com/mljs/matrix/blob/master/src/abstractMatrix.js#L1689-L1703) for a list of all supported operators and [here](https://github.com/mljs/matrix/blob/master/src/abstractMatrix.js#L1724-L1728) for general Math methods.